src/SFA.DAS.Payments.ProviderAdjustments.Service/SFA.DAS.Payments.ProviderAdjustments.Service.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Can see a few package dependency warnings for this project

![image](https://github.com/SkillsFundingAgency/das-payments-v2-provideradjustments/assets/1659084/058149cf-404c-4818-af03-7895ca588974)

Reply:
These are resolved now